customer-churn/README.md

```diff
@@ -0,0 +1,247 @@
+zenml orchestrator register cloud_orchestrator --type=kubeflow --custom_docker_base_image_name=YOUR_IMAGE
+zenml metadata-store register kubeflow_metadata_store --type=kubeflow
+zenml artifact-store register cloud_artifact_store --type=s3 --path=$PATH_TO_YOUR_BUCKET
+
+# Register the cloud stack
+zenml stack register cloud_kubeflow_stack -m kubeflow_metadata_store -a cloud_artifact_store -o cloud_orchestrator -c cloud_registry
+```
+
+3. Activate the newly created stack.
+
+```bash
+zenml stack set cloud_kubeflow_stack
+```
+
+4. Do a pipeline run
+
+```shell
+python run_kubeflow_pipeline.py
+```
+
+5. Configure Port Forwarding and Check the Kubeflow UI to see if the model is deployed and running! 🚀
+
+```bash
+kubectl --namespace kubeflow port-forward svc/ml-pipeline-ui 8080:80
+```
+
+Now, you can go to the localhost:8080 to see the UI.
+
+We can fetch the model from the kubeflow pipeline and use it in our Inference pipeline. Following diagram shows the flow of the whole pipeline:
+![cloudkubeflowstack](_assets/wholekubeflowstack.gif)
+
+## Continuous model deployment with Seldon Core
+
+While building the real-world workflow for predicting whether a customer will churn or not, you might not want to train the model once and deploy it to production. Instead, you might want to train the model and deploy it to production when something gets triggered. This is where one of our recent integration comes in, [Seldon Core](https://github.com/zenml-io/zenml/tree/main/examples/seldon_deployment).
+
+[Seldon Core](https://github.com/SeldonIO/seldon-core) is a production-grade open-source model serving platform. It packs a wide range of features built around deploying models to REST/GRPC microservices, including monitoring and logging, model explainers, outlier detectors, and various continuous deployment strategies such as A/B testing and canary deployments, and more.
+
+It also comes equipped with a set of built-in model server implementations designed to work with standard formats for packaging ML models that greatly simplify serving models for real-time inference.
+
+In this project, we build a continuous deployment pipeline that trains a model and then serves it with Seldon Core as the industry-ready model deployment tool of choice. If you are interested in learning more about Seldon Core, you can check out the [ZenML Example](https://github.com/zenml-io/zenml/tree/main/examples/seldon_deployment). Following diagram shows the flow of the whole pipeline:
+![seldondeployment](_assets/seldoncorecontinousdeployment.gif)
+
+Now, Let's start setting up our Full AWS stack to run the pipeline using Seldon Core.
+
+1. Install the Seldon Core Integration, a set of ZenML extensions that integrate with Seldon Core.
+
+```bash
+zenml integration install Seldon
+```
+
+2. Register the stack components
+
+```bash
+aws eks --region us-east-1 update-kubeconfig --name zenml-cluster --alias zenml-eks
```

Comment:
Maybe we want to give people the option to choose regions closer to them?

Reply:
yeah, I can add a note on changing that :)